app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -318,21 +319,25 @@ class SharedCallingViewModel @AssistedInject constructor(
     private suspend fun observeInCallReactions() {
         observeInCallReactionsUseCase(conversationId).collect { message ->
 
-            val sender = participantsState.senderName(message.senderUserId)?.let { name ->
-                ReactionSender.Other(name)
-            } ?: ReactionSender.Unknown
+            if (BuildConfig.IN_CALL_REACTIONS_ENABLED) {
```

Comment:
Better to make this check when we call `observeInCallReactions()`, so we don't invoke `observeInCallReactionsUseCase`